docs/man.md

```diff
@@ -65,33 +60,46 @@ Use the `qpc-tools server install` command to install and configure the Quipucor
 
 `--db-password=DB_PASSWORD`
 
-  Specifies the database password for PostgreSQL.
+  Specifies the database password for PostgreSQL.  If omitted, qpc-tools will prompt for password.
 
 `--username=SERVER_USERNAME`
 
   Sets the Quipucords server user name. Defaults to `admin`.
 
 `--password=SERVER_PASSSWORD`
 
-  Sets the Quipucords server password.
+  Sets the Quipucords server password. If omitted, qpc-tools will prompt for password.
 
 
 #### Installing the server offline
 
-If you choose the offline option to run the install command, you must do the following steps.
+Install all dependencies for the target operating system. Dependencies include:
+- **CentOS 6 and RHEL 6**
+  - [Docker 1.7.1](http://yum.dockerproject.org/repo/main/centos/6/Packages/docker-engine-1.7.1-1.el6.x86_64.rpm)
+- **CentOS 7 and RHEL 7**
```

Comment:
Could we not do like what you did below for the postgres pull:
```
- **CentOS 7, RHEL 7, and RHEL 8** 
```

Reply:
@myersCody can you explain this more?  I am not sure I understand what you are asking for.

Reply:
So on line 93, you grouped centOS 7, RHEL 7 & RHEL 8 together since they use the same command structure. I thinking you may want to do the same thing here.

Reply:
👍